services/QuillLMS/app/services/clever_integration/sign_up/school_admin.rb

```diff
@@ -26,6 +28,16 @@ def self.district_integration(auth_hash, district)
     end
   end
 
+  #TODO: remove this method
+  def self.log_import(action, auth_hash)
+    user = User.find(ENV['CLEVER_IMPORT_LOG_USER_ID'])
+    changed_attr = auth_hash.dig(:info, :user_type)
+
+    return if ChangeLog.exists?(changed_record: user, changed_attribute: changed_attr, action: action, user: user)
+
+    ChangeLog.create!(explanation: auth_hash.to_json, changed_record: user, changed_attribute: changed_attr, action: action, user: user)
```

Comment:
I might have missed this on the other PR, but do you want to raise if the Changelog isn't valid? 

it might be better to fail silently (or log an error) rather than crash, since it's just logging. (You are closer to this code, so you might have a reason to raise).

Reply:
That's a good point.   I will remove from here and the other PR.